doc/user-guide.adoc

```diff
@@ -288,6 +289,49 @@ __OR__
   - configure auto-scaling to create (and delete) instances based on resource consumption using the `autoscaling` parameter.
   - Parameters `autoscaling.maxReplicas` and `resourceConstraints.requests.cpu` MUST be specified for auto-scaling.
 
+=== Service ports
+
+Runtime Component Operator allows users to provide multiple service ports in addition to the primary service port. The primary port is exposed from the container running the application and it's values are used to configure the Route (or Ingress), Service biniding and Knative service.
```

Comment:
It'll be good to specify the parameters that are associated with the primary service port. something like: 
"The primary service port can be configured using `service.port`, `service.targetPort`, `service.portName`,  and `service.nodePort` parameters."

spelling mistake: biniding --> binding

Change "users" to "you"